ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-main.yaml

```diff
@@ -166,32 +166,28 @@ tests:
       resources:
         requests:
           cpu: 100m
-- as: tests-extension-verify
+- as: tests-extension
   run_if_changed: ^(openshift/tests-extension/)
   steps:
     test:
-    - as: tests-extension-verify
+    - as: verify
       commands: |
         cd openshift/tests-extension
         make verify
+        make build
       from: src
       resources:
         requests:
           cpu: 100m
-- as: tests-extension
+- as: olmv1-test-ext-e2e
```

Comment:
Something like `aws-techpreview-olmv1-ext`  would be more in line with https://docs.ci.openshift.org/docs/how-tos/naming-your-ci-jobs/#configuration-for-periodic-jobs